### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by muesli**
Just a bunch of nitpicky typo fixes.

### CODE REVIEW

**Review by Giako — APPROVED** [review]
LGTM, thanks for the typo fixes!

File: cmd/immuclient/immuc/login.go [hunk 1]
```diff
@@ -173,7 +173,7 @@ func (i *immuc) ChangeUserPassword(args []string) (string, error) {
 	if err := i.ImmuClient.ChangePassword(context.Background(), []byte(username), oldpass, []byte(newpass)); err != nil {
 		return "", err
 	}
-	return fmt.Sprintf("Password of %s was changed successfuly", username), nil
+	return fmt.Sprintf("Password of %s was changed successfully", username), nil
```
Comment (jeroiraz) [inline cmd/immuclient/immuc/login.go:176]: hi @muesli, probably "Password of %s was successfully changed" it's better. If so, could you please change it and we can merge your PR. Thanks a lot

Comment (muesli) [reply cmd/immuclient/immuc/login.go]: Sure thing. Done!

**Comment by jeroiraz** [comment]
thanks @muesli!
Just two things are needed before merging this PR:
- the fixes on the login message invalidated unit-test 'TestUserChangePassword' (login_test.go)
- rebase your branch with master branch

**Pull request merged**